controllers/gitopsservice_controller_test.go

```diff
@@ -500,7 +531,7 @@ func TestReconcile_InfrastructureNode(t *testing.T) {
 			Tolerations: deploymentDefaultTolerations(),
 		},
 	}
-	fakeClient := fake.NewFakeClient(gitopsService)
+	fakeClient := fake.NewFakeClient(util.NewClusterVersion("4.12.1"), gitopsService)
```

Comment:
is this required here?

Reply:
Same as the previous one, if I don't set it here the unit test fails.

Reply:
https://github.com/redhat-developer/gitops-operator/pull/398/files#r1029254556